src/ProjectTemplates/Web.ItemTemplates/content/MvcController/.template.config/template.json

```diff
@@ -0,0 +1,58 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [ "Web", "ASP.NET" ],
+  "name": "MVC Controller",
+  "generatorVersions": "[1.0.0.0-*)",
+  "description": "A MVC Controller with or without read/write actions",
+  "tags": {
+    "language": "C#",
+    "type": "item"
+  },
+  "groupIdentity": "Microsoft.AspNetCore.Mvc.MvcController",
+  "precedence": "9800",
+  "identity": "Microsoft.AspNetCore.Mvc.MvcController.8.0",
+  "shortName": "mvcController",
+  "sourceName": "HomeController",
+  "primaryOutputs": [
+    {
+      "path": "HomeController.cs"
+    }
+  ],
+  "defaultName": "HomeController",
+  "symbols": {
+    "namespace": {
+      "description": "namespace for the generated code",
+      "replaces": "MyApp.Namespace",
+      "type": "parameter"
+    },
+    "actions": {
+      "description": "create controller with read/write actions",
+      "displayName": "Add ReadWrite Actions",
+      "type": "parameter",
+      "datatype": "bool",
+      "defaultValue": "false"
+    },
+    "HostIdentifier": {
+      "type": "bind",
+      "binding": "HostIdentifier"
+    },
+    "NameIsController": {
+      "type": "computed",
+      "value": "(name == \"Controller\")"
+    }
+  },
+  "postActions": [
+    {
+      "id": "openInEditor",
+      "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "description": "Opens HomeController.cs in the editor",
```

Comment:
If I'm not mistaken the user will only see the name `HomeController` if they name it as such. Seems like an implementation detail that the user is not aware of. Should we change this to "Opens the created controller in the editor"?

cc @phenning

Reply:
sure yeah will change,